src/Task/api/controllers/getTask/GetTaskbyTaskIDController.ts

```diff
@@ -0,0 +1,21 @@
+import { Request, Response } from 'express';
+import { GetTaskByTaskIDUseCase } from '../../../useCases/getTaskByID/GetTaskByTaskIDUseCase';
+
+export class GetTaskbyTaskIDController {
+  constructor(
+    private readonly getTaskByTaskIDUseCase: GetTaskByTaskIDUseCase,
+  ) {}
+
+  async execute(request: Request, response: Response): Promise<void> {
+    const taskID = request.query.taskId as string;
```

Comment:
this should be request.params.taskId to retrieve taskId from url "tasks/:taskId"
you can check here
https://stackoverflow.com/questions/14417592/node-js-difference-between-req-query-and-req-params

Reply:
I have fixed :)